libs/packages/components/src/lib/autocomplete-search/autocomplete-search.component.ts

```diff
@@ -347,6 +360,83 @@ export class SDSAutocompleteSearchComponent implements ControlValueAccessor {
     }
   }
 
+  private onArrowGroupDown(): void {
+    if (this.results && this.results.length > 0) {
+      if (this.highlightedIndex <= this.results.length - 1) {
+        let childLength = this.results[this.highlightedIndex][
+          this.configuration.groupByChild
+        ].length;
+        if (this.highlightedChildIndex === childLength) {
+          this.highlightedIndex++;
+          this.highlightedChildIndex = 0;
+          if (this.configuration.isSelectableGroup) {
+            this.setHighlightedItem(this.results[this.highlightedIndex]);
+          } else {
+            this.setHighlightedItem(
+              this.results[this.highlightedIndex][
+                this.configuration.groupByChild
+              ][this.highlightedChildIndex]
+            );
+            this.highlightedChildIndex++;
+          }
+        } else {
+          this.setHighlightedItem(
+            this.results[this.highlightedIndex][
+              this.configuration.groupByChild
+            ][this.highlightedChildIndex++]
+          );
+        }
+      }
+      this.scrollSelectedItemIntoView();
```

Comment:
Getting an error when reaching the bottom of the grouped list:

main.364c2f9b85ae0b7fd30b.js:1 ERROR TypeError: Cannot read property 'getElementsByTagName' of undefined
    at t.scrollSelectedItemIntoView (1.caf8d05b334313f87ba3.js:1)
    at t.onArrowGroupDown (1.caf8d05b334313f87ba3.js:1)
    at t.onKeydown (1.caf8d05b334313f87ba3.js:1)
    at Object.handleEvent (1.caf8d05b334313f87ba3.js:1)
    at Object.handleEvent (main.364c2f9b85ae0b7fd30b.js:1)
    at Object.handleEvent (main.364c2f9b85ae0b7fd30b.js:1)
    at wr (main.364c2f9b85ae0b7fd30b.js:1)
    at main.364c2f9b85ae0b7fd30b.js:1
    at HTMLInputElement.<anonymous> (main.364c2f9b85ae0b7fd30b.js:1)
    at t.invokeTask (polyfills.a89bac0a374016bbb079.js:1)

Reply:
Also, there is a behavior where if you keyboard below a group, and then hit up arrow, it stops highlighting visually. The item appears to be getting selected because the scroll into view is still working.